### PR TITLE
Added pipeline yaml files

### PR DESCRIPTION
**Pull request by Vitals9367**

### CODE REVIEW

File: azure-pipelines-stageprod.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Continuous integration (CI) triggers cause a pipeline to run whenever you push 
+# an update to the specified branches or you push specified tags.
+trigger:
+  tags:
+    include:
+    - release-*tags/*
```
Comment (charn) [inline azure-pipelines-stageprod.yml:6]: Any reason why this is different in the UI related repos i.e. why is this not `- release-*`?

Comment (Vitals9367) [reply azure-pipelines-stageprod.yml:6]: Oh, my error. It has to be - release-*

Comment (Vitals9367) [reply azure-pipelines-stageprod.yml]: Should be fixed now.

**Review by nikomakela — APPROVED** [review]
looks good to me, but I have no experience with these

**Comment by nikomakela** [comment]
Is there a good reason why the builds fails?

**Pull request merged**